vispy/gloo/program.py

```diff
@@ -301,7 +301,10 @@ def __setitem__(self, name, data):
             elif kind == 'attribute':
                 # Is this a constant value per vertex
                 is_constant = False
-                isscalar = lambda x: isinstance(x, (float, int))
+
+                def isscalar(x):
+                    return isinstance(x, (float, int))
+
```

Comment:
why the F are lambdas bad according to flake? I personally find the lambda more pretty (and shorter!) in this case.

Perhaps also don't apply this rule?